### PR TITLE
Avoid using `tigris` and use the existing geometry in the dataset (to fix the rendering issue)

### LINKED ISSUE #122: Some census tracts do not show up on Shinyapps.io (but they do on local dev)
The deployed app on Shinyapps.io is not showing census tracts on the map:

(left = deployed; right = local)

image

### PR DESCRIPTION
**Pull request by nsunami**
This PR will restructure the codes for the map generation to hopefully fix the issue of census tracts not plotting on shinyapps.io (fix #122)

Creating a PR to deploy to the test app.

### CODE REVIEW

**Comment by nsunami** [comment]
@mehak25 I removed using tigris here (+ some adjustments to optimize the rendering via using functions). I think it is fixed, but I want you to review this before merging! 🤞

**Comment by mehak25** [comment]
It seems to be working. You can merge it. Thanks.

**Comment by nsunami** [comment]
Thanks so much @mehak25!

I noticed that the output table is not sorted in ascending order. I will fix it in this PR and merge it once I'm done.

File: app/scripts/advocates.R [hunk 1]
```diff
@@ -3,40 +3,37 @@ library(tidyverse)
...
+    mutate(prop_above30 = (rent_above30/tot_hh)*100,
          prop_above50 = (rent_above50/tot_hh)*100,
          prop_reported_HUD = (reported_HUD/tot_hh)*100) %>%
-  mutate_at(vars(prop_above30),~ round(., 2)) %>%
-  mutate_at(vars(prop_above50),~ round(., 2)) %>%
-  mutate_at(vars(prop_reported_HUD),~ round(., 2)) %>%
-  mutate_at(vars(reported_HUD),as.integer) %>%
-  mutate_at(vars(rent_above30),as.integer) %>%
-  mutate_at(vars(rent_above50),as.integer) %>%
+  mutate_at(vars(prop_above30,
```
Comment (nsunami) [inline app/scripts/advocates.R:30]: Just to note that `mutate_at` is superseded by `across` right now.

Comment (nsunami) [reply app/scripts/advocates.R:30]: I'll leave them as-is in this PR, but it would be a good idea to move towards across in the future.

**Pull request merged**